server/minify.mjs

```diff
@@ -9,73 +10,73 @@ import uglifyES from '@node-minify/uglify-es';
+                'client/css/widgets/classes.css',
+                'client/css/widgets/deck.css',
+                'client/css/widgets/holder.css',
+                'client/css/widgets/label.css',
+                'client/css/widgets/pile.css',
+                'client/css/widgets/spinner.css'
+            ],
+            output: os.tmpdir() + '/out.css'
+        }).then(function(min) {
+            roomHTML = roomHTML.replace(/ \{\{CSS\}\} /, min);
+            return minify({
+                compressor: process.env.NOCOMPRESS ? noCompress : uglifyES,
+                input: [
+                    'client/js/domhelpers.js',
+                    'client/js/connection.js',
+                    'client/js/serverstate.js',
+                    'client/js/editmode.js',
+                    'client/js/geometry.js',
+                    'client/js/mousehandling.js',
+                    'client/js/statemanaged.js',
+                    'client/js/widgets/widget.js',
 
-          'client/js/widgets/basicwidget.js',
-          'client/js/widgets/button.js',
-          'client/js/widgets/card.js',
-          'client/js/widgets/deck.js',
-          'client/js/widgets/holder.js',
-          'client/js/widgets/label.js',
-          'client/js/widgets/pile.js',
-          'client/js/widgets/spinner.js',
+                    'client/js/widgets/basicwidget.js',
+                    'client/js/widgets/button.js',
+                    'client/js/widgets/card.js',
+                    'client/js/widgets/deck.js',
+                    'client/js/widgets/holder.js',
+                    'client/js/widgets/label.js',
+                    'client/js/widgets/pile.js',
+                    'client/js/widgets/spinner.js',
 
-          'client/js/overlays/players.js',
-          'client/js/overlays/states.js',
+                    'client/js/overlays/players.js',
+                    'client/js/overlays/states.js',
 
-          'client/js/main.js'
-        ],
-        output: '/tmp/out.js'
-      });
-    }).then(function(min) {
-      return minify({
-        compressor: htmlMinifier,
-        content: roomHTML.replace(/ \{\{JS\}\} /, min),
-        options: {
-          conservativeCollapse: true
-        }
-      })
-    }).then(function(min) {
-      roomHTML = min;
-      zlib.gzip(min, (err, gzipped) => {
-        resolve({ min, gzipped });
-      });
+                    'client/js/main.js'
+                ],
+                output: os.tmpdir() + '/out.js'
+            });
+        }).then(function(min) {
+            return minify({
+                compressor: htmlMinifier,
+                content: roomHTML.replace(/ \{\{JS\}\} /, min),
+                options: {
+                    conservativeCollapse: true
+                }
+            })
+        }).then(function(min) {
+            roomHTML = min;
+            zlib.gzip(min, (err, gzipped) => {
+                resolve({ min, gzipped });
+            });
+        });
     });
-  });
-}
+}
```

Comment:
please restore the missing newline. And try to configure your editor to include newlines on the last line in text files.